Pod/Scripts/ArtsySetup.rb

```diff
@@ -1,13 +1,22 @@
 #!/usr/bin/env ruby
 
-puts "Trying to clone Artsy's Private fonts, it is OK if it fails."
+puts "Trying to clone Artsy's Private fonts, it is OK if it fails for OSS contributors."
+
+def raise_if_hokusai!
+  `which hokusai`
+  if $?.exitstatus == 0
+    raise "You appear to have Hokusai installed but we couldn't access Artsy's private fonts repository. Failing pod install. Ask the #front-end-ios Slack channel for help."
```

Comment:
If I understand this PR correct, the Hokusai check is really an implicit check to see if you’re a staff member, right? In that case I think this message should focus a little more on that, otherwise it might just distract. Eg:

```suggestion
    raise "You appear to have Hokusai installed, which likely means you’re an Artsy staff member—however, we couldn't access Artsy's private fonts repository. Failing pod install. Ask the #front-end-ios Slack channel for help."
```

Reply:
Yeah exactly. Good suggestion 👍